### PR TITLE
Set english language selected by default

### LINKED ISSUE #2691: Fix the language popover
**Actual Behaviour**
When you de-select all the languages it  displays "-" and  popover  is not closed .

**Expected Behaviour**
When you de-select all the language , by default it should display English and close the popover.

**Screenshots of the issue**
Screenshot

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #2691 

Changes: When all the languages are deselected it displays english as default language

Demo Link: https://pr-2921-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by Dilshaad21** [comment]
Some issues with my base branch will make a new pr

**Comment by Dilshaad21** [comment]
@akshatnitd PTAL.

File: src/components/NavigationBar/index.js [hunk 1]
```diff
@@ -327,6 +327,10 @@ class NavigationBar extends Component {
 
   handleLanguageChange = (event, index, values) => {
     localStorage.setItem('languages', event.target.value);
+    let languages = [...event.target.value];
+    if (languages.length === 0) {
+      event.target.value.push('en');
```
Comment (akshatnitd) [inline src/components/NavigationBar/index.js]: Do not push something inside the event. Instead, push into the `languages` variable.

File: src/components/NavigationBar/index.js [hunk 2]
```diff
@@ -327,8 +327,12 @@ class NavigationBar extends Component {
 
   handleLanguageChange = (event, index, values) => {
     localStorage.setItem('languages', event.target.value);
+    let languages = [...event.target.value];
```
Comment (akshatnitd) [inline src/components/NavigationBar/index.js]: Why using spread operator here?
Can' t we just do, 
```
let languages = event.target.value;
```

Comment (Dilshaad21) [reply src/components/NavigationBar/index.js]: Ok @akshatnitd I agree , will make the changes.

**Pull request merged**